### PR TITLE
Add safeSeekOffset to StreamingConfiguration (#1723)

### PR DESCRIPTION
**Pull request by mr-tichyt**
Please see the #1723 for more details.
Thank you

### CODE REVIEW

File: CONTRIBUTORS [hunk 1]
```diff
@@ -74,6 +74,7 @@ Thomas Stephens <thomas@ustudio.com>
 Tim Plummer <objelisks@google.com>
 Timothy Drews <tdrews@google.com>
 Tomas Bucher <collapsingtesseract@gmail.com>
+Tomas Tichy <mr.tichyt@gmail.com>
```
Comment (joeyparrish) [inline CONTRIBUTORS]: Please add yourself or your company (as appropriate) to AUTHORS as well.  Thanks!

File: docs/tutorials/config.md [hunk 2]
```diff
@@ -46,6 +46,7 @@ player.getConfiguration();
        defaultBandwidthEstimate: 500000
        enabled: true
        restrictions: Object
+       safeSeekOffset: 5
```
Comment (joeyparrish) [inline docs/tutorials/config.md]: I think this does not belong under "abr".  You seem to have another copy under "streaming".

File: lib/media/playhead.js [hunk 3]
```diff
@@ -353,11 +355,10 @@ shaka.media.Playhead.prototype.reposition_ = function(currentTime) {
...
   // on the next call and we would loop forever.
-  //
-  // Offset by 5 seconds since Chromecast takes a few seconds to start playing
-  // after a seek, even when buffered.
-  let seekStart = this.timeline_.getSafeSeekRangeStart(5);
-  let seekSafe = this.timeline_.getSafeSeekRangeStart(rebufferingGoal + 5);
+  let seekStart = this.timeline_.getSafeSeekRangeStart(safeSeekOffset);
+  let seekSafe = this.timeline_.getSafeSeekRangeStart(
+      rebufferingGoal + safeSeekOffset
+  );
```
Comment (joeyparrish) [inline lib/media/playhead.js]: style nit: please move this to the previous line

File: docs/tutorials/config.md (hunk 2 above)
Comment (mr-tichyt) [reply docs/tutorials/config.md]: my bad

**Comment by mr-tichyt** [comment]
Hi, all three fixed. I amended the commit to keep clear history if that is OK... Have a nice weekend

**Pull request merged**